imgcreate/creator.py

```diff
@@ -1,9 +1,10 @@
 #
 # creator.py : ImageCreator and LoopImageCreator base classes
 #
-# Copyright 2007, Red Hat  Inc.
+# Copyright 2007, Red Hat, Inc.
 # Copyright 2016, Kevin Kofler
 # Copyright 2016, Neal Gompa
+# Copyright 2017, Fedora Project
```

Comment:
It was intentional to assign the rights for the imgcreate components to the Fedora Project.